build.sbt

```diff
@@ -193,15 +190,15 @@ lazy val common = crossProject(allPlatforms: _*).in(file("scalameta/common")).se
   enableMacros,
   buildInfoPackage := "scala.meta.internal",
   buildInfoKeys := Seq[BuildInfoKey](version),
-  crossScalaVersions := AllScalaVersions
+  crossScalaVersions := EarliestScalaVersions
```

Comment:
so common, trees, parsers, and scalameta are all built for the oldest version per major? assuming that older minor will work with newer minor library? and only semantic is building newer ones?

Reply:
> assuming that older minor will work with newer minor library

Yes, since the scala library is still binary compatible.